tests/test_preprocess.py

```diff
@@ -32,3 +41,72 @@ def test_process_exception(tmpdir):
             erosion_kernel_type,
             erosion_kernel_size,
         )
+
+
+@pytest.fixture
+def batch_process_data_directory(
+    tmp_path, date="2012-08-01", day=214, satellite="terra"
+):
+    test_dir = os.path.dirname(__file__)
+    shutil.copyfile(
+        Path(f"{test_dir}/input/reproj_land.tiff"), tmp_path / Path("landmask.tiff")
+    )
+    shutil.copyfile(
+        Path(f"{test_dir}/input/tci/tci_{date}_{day}_{satellite}.tiff"),
+        tmp_path / Path("truecolor.tiff"),
+    )
+    shutil.copyfile(
+        Path(f"{test_dir}/input/cloud/cloud_{date}_{day}_{satellite}.tiff"),
+        tmp_path / Path("cloud.tiff"),
+    )
+    logger.debug(f"initialized data in {tmp_path}")
+    return tmp_path
+
+
+@pytest.fixture
+def process_data_directory(tmp_path):
+    test_dir = os.path.dirname(__file__)
+    shutil.copyfile(
+        Path(f"{test_dir}/process/landmask.tiff"), tmp_path / Path("landmask.tiff")
+    )
+    shutil.copyfile(
+        Path(f"{test_dir}/process/truecolor.tiff"), tmp_path / Path("truecolor.tiff")
+    )
+    shutil.copyfile(
+        Path(f"{test_dir}/process/cloud.tiff"), tmp_path / Path("cloud.tiff")
+    )
+    logger.debug(f"initialized data in {tmp_path}")
+    return tmp_path
```

Comment:
Here you can do without `os`.
```suggestion
@pytest.fixture
def process_data_directory(tmp_path):
    test_dir = Path(__file__).parent
    shutil.copyfile(
        test_dir / "process/landmask.tiff", tmp_path / "landmask.tiff"
    )
    shutil.copyfile(
        test_dir / "process/truecolor.tiff", tmp_path / "truecolor.tiff"
    )
    shutil.copyfile(
        test_dir / "process/cloud.tiff", tmp_path / "cloud.tiff"
    )
    logger.debug(f"initialized data in {tmp_path}")
    return tmp_path
```